.eslintrc.js

```diff
@@ -56,6 +56,9 @@ module.exports = {
         "@typescript-eslint/triple-slash-reference": "error",
         "@typescript-eslint/type-annotation-spacing": "error",
         //"@typescript-eslint/unified-signatures": "error",
+        "@typescript-eslint/no-floating-promises": "error",
```

Comment:
`no-floating-promises` doesn't seem to be necessary. the code is not handling the values that are returned from `vscode.window.showInformationMessage` or `vscode.window.showErrorMessage`. You can remove this.

Reply:
The important ones are in state.ts, and those had caused a number of troubles :)